Makefile

```diff
@@ -600,15 +590,14 @@ copy-go-binaries-to-image-dir:
 	cp bin/linux_$(GOARCH)/central image/bin/central
 ifdef CI
```

Comment:
Nit: I would prefer to use `ifndef CI ... else ... fi` here, to match the shape of the `cli-build` target